kalite/coachreports/static/js/coachreports/student_progress.js

```diff
@@ -94,6 +94,7 @@ var PlaylistProgressView = Backbone.View.extend({
         this.$(".playlist-progress-details").slideToggle();
     }
 });
+var count=0;  //A variable to keep a count of the number of reports of quizes/videos that would be present in the student's progress report.
```

Comment:
You should remove this. It's polluting the global namespace, and you don't even use it.